### PR TITLE
fixed cpplint errors

### PR DESCRIPTION
**Pull request by StrikerRUS**
Refer to #1990.

### CODE REVIEW

File: src/lightgbm_R.cpp [hunk 1]
```diff
@@ -9,15 +9,15 @@
 #include <LightGBM/utils/openmp_wrapper.h>
 #include <LightGBM/utils/text_reader.h>
 
+#include <R_ext/Rdynload.h>
+
```
Comment (jameslamb) [inline src/lightgbm_R.cpp:13]: ah ok, thanks! I should have looked at `cpplint` manually in the logs when I did #2911

**Comment by StrikerRUS** [comment]
@jameslamb Thanks!

>  I Do see failures on Azure DevOps that we'll have to investigate 

Random network error during loading SWIG.

```
Invoke-WebRequest : Oops, looks like something went wrong.
```

File: src/lightgbm_R.cpp (hunk 1 above)
Comment (StrikerRUS) [reply src/lightgbm_R.cpp:13]: I hope soon we will not have to do this.

Comment (jameslamb) [reply src/lightgbm_R.cpp:13]: +1 I agree

**Pull request merged**